app/assets/templates/home.html

```diff
@@ -1,5 +1,8 @@
-<div><header></header></div>
+<header></header>
```

Comment:
It's a directive but i should be renamed **asap** because it interferes with the html tag `<header>`